Revit_Core_Engine/Convert/Revit/FromRevit/View.cs

```diff
@@ -0,0 +1,135 @@
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using Autodesk.Revit.DB;
+using BH.Engine.Adapters.Revit;
+using BH.oM.Adapters.Revit.Elements;
+using BH.oM.Adapters.Revit.Settings;
+using BH.oM.Base;
+using BH.oM.Base.Attributes;
+using BH.oM.Revit.Enums;
+using BH.oM.Revit.Views;
+using BH.Revit.Engine.Core;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Drawing;
+using System.Linq;
+using System.Security.Policy;
+using System.Xml.Linq;
+using FilterRule = BH.oM.Revit.FilterRules.FilterRule;
+using OverrideGraphicSettings = BH.oM.Adapters.Revit.Elements.OverrideGraphicSettings;
+using View = Autodesk.Revit.DB.View;
+
+namespace BH.Revit.Engine.Core
+{
+    public static partial class Convert
+    {
+        /***************************************************/
+        /****               Public Methods              ****/
+        /***************************************************/
+
+        [Description("Converts a Revit View to BH.oM.Adapters.Revit.Elements.View.")]
+        [Input("revitView", "Revit View to be converted.")]
+        [Input("settings", "Revit adapter settings to be used while performing the convert.")]
+        [Input("refObjects", "Optional, a collection of objects already processed in the current adapter action, stored to avoid processing the same object more than once.")]
+        [Output("view", "BH.oM.Adapters.Revit.Elements.View resulting from converting the input Revit View.")]
+        public static oM.Adapters.Revit.Elements.View ViewFromRevit(this View revitView, RevitSettings settings = null, Dictionary<string, List<IBHoMObject>> refObjects = null)
+        {
+            settings = settings.DefaultIfNull();
+
+            oM.Adapters.Revit.Elements.View view = refObjects.GetValue<oM.Adapters.Revit.Elements.View>(revitView.Id.IntegerValue);
+            if (view != null)
+                return view;
+
+            /*1. Transfer ViewFilters and corresponding OverrideGraphicSettings into ViewFilterWithOverrides Objects - via STREAMS */
+            List<ViewFilterWithOverrides> filtersWithOverrides = revitView.GetFilters().ToDictionary<ElementId, ViewFilter, OverrideGraphicSettings>
```

Comment:
Some view types (e.g. project browser) do not support overrides, causing the code to crash in case of a simple pull:
![image](https://github.com/user-attachments/assets/037a53c8-d9da-4a17-a952-fd00cdb2d153)

Reply:
@pawelbaran, thanks for flagging this.
I've added an  if statement on the boolean parameter "AreGraphicsOverridesAllowed()" of the Autodesk View Class.
Whenever the parameter is true, the method GetFilters() gets run and no error messages get triggered.
Whenever the parameter is false, the corresponding BHoM view gets created without any filters assigned to it.

![image](https://github.com/user-attachments/assets/0c105807-89f0-41e9-84ec-8c3fe256f4de)